Gravity-Notes/src/Pages/Homepage/Homepage.css

```diff
@@ -74,7 +74,7 @@
     }    
  }
 
- @media screen and (max-width:430px) {
+ @media screen and (max-width:429px) {
```

Comment:
use rems for such big values

Reply:
its media screen it should always be in px ig